components/styled-links.js

```diff
@@ -58,14 +30,36 @@ export const Anchor = ({href, prefetch, external, children}) => { // eslint-disa
   )
 }
 
-export const Button = ({href, children}) => { // eslint-disable-line no-unused-vars
-  return (
-    <StyledButton href={href}>{children}</StyledButton>
-  )
+const solidColors = {
+  backgroundColor: colors.primary,
+  color: 'white'
 }
 
-export const SecondaryButton = ({href, children}) => { // eslint-disable-line no-unused-vars
-  return (
-    <StyledSecondaryButton href={href}>{children}</StyledSecondaryButton>
-  )
+const transparentColors = {
+  backgroundColor: 'white',
+  color: colors.primary
 }
+
+const secondaryButtonStyles = {
+  ...transparentColors,
+  ':hover': solidColors
+}
+
+export const Button = glamorous.a(
+  basicLinkStyles,
+  {
+    fontSize: '1em',
+    border: `1px solid ${colors.primaryMed}`,
+    padding: '0.25em 1em',
+    textDecoration: 'none',
+    borderRadius: 4,
+    display: 'inline-block',
+    margin: '1em',
+    transition: 'all .3s',
+    ...solidColors,
+    ':hover': transparentColors
+  },
+  props => ({
```

Comment:
Oh, also, instead of doing object spread to merge them (which is a shallow merge), we could just make each of these a totally separate parameter and glamor will merge them for us :+1: